app/src/main/java/com/example/taskyapplication/agenda/task/presentation/screens/TaskDetailScreen.kt

```diff
@@ -0,0 +1,134 @@
+package com.example.taskyapplication.agenda.task.presentation.screens
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.material.Icon
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.outlined.CheckCircle
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.example.taskyapplication.TaskyBaseScreen
+import com.example.taskyapplication.agenda.data.model.AgendaItemType
+import com.example.taskyapplication.agenda.data.model.ReminderOptions
+import com.example.taskyapplication.agenda.presentation.components.AgendaDescriptionText
+import com.example.taskyapplication.agenda.presentation.components.AgendaIconTextRow
+import com.example.taskyapplication.agenda.presentation.components.AgendaItem
+import com.example.taskyapplication.agenda.presentation.components.AgendaItemDateTimeRow
+import com.example.taskyapplication.agenda.presentation.components.AgendaItemDeleteTextButton
+import com.example.taskyapplication.agenda.presentation.components.AgendaTitleRow
+import com.example.taskyapplication.agenda.presentation.components.DetailScreenHeader
+import com.example.taskyapplication.agenda.presentation.components.ReminderTimeRow
+import com.example.taskyapplication.agenda.task.presentation.TaskUiState
+import com.example.taskyapplication.ui.theme.TaskyDesignSystem.Companion.taskyColors
+import com.example.taskyapplication.ui.theme.TaskyTypography
+
+@Composable
+fun TaskDetailScreen(
+    modifier: Modifier = Modifier,
+    screenHeading: @Composable () -> Unit = {},
+    agendaItem: AgendaItemType,
```

Comment:
Why do you need to pass an `AgendaItemType` here if you know you're dealing with a Task?